### PR TITLE
Update/SOF-7291 add enum

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: src/other/iframe-messaging/IframeToFromHostMessageHandler.ts [hunk 1]
```diff
@@ -6,8 +6,18 @@ type HandlersMap = {
...
+export enum ActionEnum {
+    GetData = "get-data",
+    SetData = "set-data",
+    Info = "info",
+}
+
 class IframeToFromHostMessageHandler {
-    private handlers: HandlersMap = { "get-data": [], "set-data": [], info: [] };
+    private handlers: HandlersMap = {
+        [ActionEnum.GetData]: [],
```
Comment (timurbazhirov) [inline src/other/iframe-messaging/IframeToFromHostMessageHandler.ts]: Let's use lowercase first letter and use enums from the IframeMessageSchema

File: dist/other/iframe-messaging/IframeToFromHostMessageHandler.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
+import { Action as ActionEnum, Type as TypeEnum, } from "@mat3ra/esse/dist/js/types";
```
Comment (k0stik) [inline dist/other/iframe-messaging/IframeToFromHostMessageHandler.js:1]: This is the issue

Comment (k0stik) [reply dist/other/iframe-messaging/IframeToFromHostMessageHandler.js:1]: You can not import *.ts in *.js file

Comment (VsevolodX) [reply dist/other/iframe-messaging/IframeToFromHostMessageHandler.js:1]: But this is dist that was generated

Comment (k0stik) [reply dist/other/iframe-messaging/IframeToFromHostMessageHandler.js:1]: Probably this thing expects us to convert types.ts to types.js as well

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**